packages/docutils/requirements.txt

```diff
@@ -0,0 +1,4 @@
+mkdocs==1.4.0
```

Comment:
shall we rather switch to Poetry or Pipenv for deps management?

Reply:
My Python knowledge is from like a decade ago, so whatever you think.

Reply:
I would advice to use pipenv